internal/service/loadbalancer/manual_certificate_bundle.go

```diff
@@ -5,157 +5,219 @@ import (
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-func ResourceManualCertificateBundle() *schema.Resource {
-	return &schema.Resource{
-		Description:   "This resource represents manual certificate bundle",
-		CreateContext: resourceManualCertificateBundleCreate,
-		ReadContext:   resourceManualCertificateBundleRead,
-		UpdateContext: resourceManualCertificateBundleUpdate,
-		DeleteContext: resourceCertificateBundleDelete,
-		Importer: &schema.ResourceImporter{
-			StateContext: schema.ImportStatePassthroughContext,
-		},
-		Schema: map[string]*schema.Schema{
-			"name": {
-				Description:      "The name of the bundle must be unique within customer account.",
-				Type:             schema.TypeString,
-				Required:         true,
-				ValidateDiagFunc: validateNameDiagFunc,
+var (
+	_ resource.Resource                = &manualCertificateBundleResource{}
+	_ resource.ResourceWithConfigure   = &manualCertificateBundleResource{}
+	_ resource.ResourceWithImportState = &manualCertificateBundleResource{}
+)
+
+func NewManualCertificateBundleResource() resource.Resource {
+	return &manualCertificateBundleResource{}
+}
+
+type manualCertificateBundleResource struct {
+	client *service.Service
+}
+
+func (r *manualCertificateBundleResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_loadbalancer_manual_certificate_bundle"
+}
+
+// Configure adds the provider configured client to the resource.
+func (r *manualCertificateBundleResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	r.client, resp.Diagnostics = utils.GetClientFromProviderData(req.ProviderData)
+}
+
+type manualCertificateBundleModel struct {
+	Certificate      types.String `tfsdk:"certificate"`
+	ID               types.String `tfsdk:"id"`
+	Intermediates    types.String `tfsdk:"intermediates"`
+	Name             types.String `tfsdk:"name"`
+	NotAfter         types.String `tfsdk:"not_after"`
+	NotBefore        types.String `tfsdk:"not_before"`
+	OperationalState types.String `tfsdk:"operational_state"`
+	PrivateKey       types.String `tfsdk:"private_key"`
+}
+
+func (r *manualCertificateBundleResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "This resource represents manual certificate bundle",
+		Attributes: map[string]schema.Attribute{
+			"certificate": schema.StringAttribute{
+				MarkdownDescription: "Certificate within base64 string must be in PEM format.",
```

Comment:
Would this be easier to understand? (Same issue also in API docs and `intermediates`)

```suggestion
				MarkdownDescription: "Certificate as base64 encoded string. Must be in PEM format.",
```

Reply:
Changed the description(s) accordingly